cmd/tink-cli/cmd/get/get.go

```diff
@@ -0,0 +1,100 @@
+package get
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/jedib0t/go-pretty/table"
+	"github.com/spf13/cobra"
+)
+
+type CmdOpt struct {
+	// Headers is the list of headers you want to print as part of the list
+	Headers []string
+	// RetrieveData reaches out to Tinkerbell and it gets the required data
+	RetrieveData func(context.Context) ([]interface{}, error)
+	// PopulateTable populates a table with the data retrieved with the RetrieveData function.
+	PopulateTable func([]interface{}, table.Writer) error
+
+	// Output specifies the output you want the list of resources printed
+	// out. By default it is table but it can be JSON ar CSV.
+	Output string
+	// NoHeaders does not print the header line
+	NoHeaders bool
+}
+
+const shortDescr = `
```

Comment:
this initial new line makes it so that the short description messes with the help output:
```
./tink-cli hardware --help
tink hardware client
...
Available Commands:
  delete      delete hardware by id
  get         
Display one or many resources
  id          get hardware by id
  ip          get hardware by any associated ip
...
```
If you remove this new line then it looks correct:
```
./tink-cli hardware --help
tink hardware client
...
Available Commands:
  delete      delete hardware by id
  get         Display one or many resources
  id          get hardware by id
  ip          get hardware by any associated ip
...
```

Reply:
and I just noticed that its got a capital letter when it should be lower case actually